### PR TITLE
bug: commas on deploy action triggers

### PR DESCRIPTION
**Pull request by DerekRoberts**
I introduced deploy triggers with commas that shouldn't have been there.  This action removes them.  Please expect a few commits to be made/removed during testing.

Thanks to @RMCampos for catching this and troubleshooting.  :)

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-spar-296-backend.apps.silver.devops.gov.bc.ca/)
[Frontend](https://nr-spar-296-frontend.apps.silver.devops.gov.bc.ca/)
[Oracle-API](https://nr-spar-296-oracle-api.apps.silver.devops.gov.bc.ca/)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

**Comment by DerekRoberts** [comment]
Testing works!  Look at run times to see whether a job triggered.  They complete successfully whether triggered or not.  Jobs fail only on actual failures.

Not triggered:
No changes

Triggered:
Yes changes

File: .github/workflows/pr-open.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ jobs:
             -p ORACLE_DB_USER=${{ secrets.DB_USER }}
             -p ORACLE_DB_PASSWORD='${{ secrets.DB_PASSWORD }}'
             -p FORESTCLIENTAPI_KEY='${{ secrets.FORESTCLIENTAPI_KEY }}'
-          triggers: ('common/', 'backend/', 'frontend/', 'oracle-api/')
+          triggers: ('common/' 'backend/' 'frontend/' 'oracle-api/')
```
Comment (RMCampos) [inline .github/workflows/pr-open.yml:52]: 👍🏻 sounds good. But we should update all triggers in this file, right?

**Review by RMCampos — CHANGES_REQUESTED** [review]

File: .github/workflows/pr-open.yml (hunk 1 above)
Comment (RMCampos) [reply .github/workflows/pr-open.yml:52]: Oh, no more triggers with comma! Good job then! Approving now!

**Review by RMCampos — APPROVED** [review]
Great job! 💯

**Pull request merged**